apps/admin/src/pages/hackathons/displayHackathons.tsx

```diff
@@ -0,0 +1,38 @@
+import { DataTable } from "~/components/ui/data-table";
+import { trpc } from "~/trpc";
+import { columns } from "./hackathon-columns";
+
+export function displayHackathons() {
```

Comment:
This function doesn't look its very useful since its just an additional wrapper around `HackathonsTable`. You should just use HackathonsTable directly wherever this is being used.